sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -193,6 +184,21 @@ export class ApplicationStudentsController extends BaseController {
       // then study start date taken from offering
       studyStartDate = offering.studyStartDate;
       studyEndDate = offering.studyEndDate;
+      // This ensures that if an offering is selected in student application,
```

Comment:
I am not able to follow the comment. I got the idea but it seems to be passing the wrong information.